lib/cylc/gui/gscan.py

```diff
@@ -485,8 +485,8 @@ def __init__(self, hosts=None, owner=None, poll_interval=None):
             self.warn_icon_grey, 0, False)  # b&w warn icon pixbuf
         status_column.pack_start(warn_icon, expand=False)
         status_column.set_cell_data_func(warn_icon, self._set_error_icon_state)
-        warn_icon.set_property('visible', False)
-
+        self.warn_icon_blank = gtk.gdk.Pixbuf(  # Transparent pixbuff.
+            gtk.gdk.COLORSPACE_RGB, True, 8, 17, 17).fill(0x00000000)
```

Comment:
Tis the width/height of the pixbuf icons normally implied from the xpm strings. I'll stick it in a constant here for this module.